tests/kernel/install_ltp.pm

```diff
@@ -196,3 +196,5 @@ will be used.
 Overrides the official LTP GitHub repository URL.
 
 =cut
+
+# vim: set sw=4 et:
```

Comment:
why this commit?

Reply:
I don't particularly mind, but if people started adding this for other editors then it would become a mess.

Otherwise looks good to me.